packages/core/src/toggle/toggle.element.scss

```diff
@@ -1,11 +1,11 @@
-// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
 @import './../styles/tokens/generated/index';
 
 :host {
-  --background: #{$cds-alias-status-neutral-tint};
+  --background: #{$cds-global-color-gray-400};
```

Comment:
Does this mean that the background always be this color (in dark theme and other themes)?

Reply:
Fixed, I think that was from testing various contrast levels, it should of been a updated status alias color